dcm4che-net-audit/src/test/java/org/dcm4che3/net/audit/AuditLoggerTest.java

```diff
@@ -0,0 +1,356 @@
+/* ***** BEGIN LICENSE BLOCK *****
+ * Version: MPL 1.1/GPL 2.0/LGPL 2.1
+ *
+ * The contents of this file are subject to the Mozilla Public License Version
+ * 1.1 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * http://www.mozilla.org/MPL/
+ *
+ * Software distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
+ * for the specific language governing rights and limitations under the
+ * License.
+ *
+ * The Original Code is part of dcm4che, an implementation of DICOM(TM) in
+ * Java(TM), hosted at https://github.com/dcm4che.
+ *
+ * The Initial Developer of the Original Code is
+ * Agfa Healthcare.
+ * Portions created by the Initial Developer are Copyright (C) 2022
+ * the Initial Developer. All Rights Reserved.
+ *
+ * Contributor(s):
+ * See @authors listed below
+ *
+ * Alternatively, the contents of this file may be used under the terms of
+ * either the GNU General Public License Version 2 or later (the "GPL"), or
+ * the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
+ * in which case the provisions of the GPL or the LGPL are applicable instead
+ * of those above. If you wish to allow use of your version of this file only
+ * under the terms of either the GPL or the LGPL, and not to allow others to
+ * use your version of this file under the terms of the MPL, indicate your
+ * decision by deleting the provisions above and replace them with the notice
+ * and other provisions required by the GPL or the LGPL. If you do not delete
+ * the provisions above, a recipient may use your version of this file under
+ * the terms of any one of the MPL, the GPL or the LGPL.
+ *
+ * ***** END LICENSE BLOCK ***** */
+package org.dcm4che3.net.audit;
+
+import java.io.File;
+import java.util.Collections;
+
+import org.dcm4che3.net.Connection;
+import org.dcm4che3.net.Device;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Marryat Ma
+ */
+public class AuditLoggerTest {
+
+    private AuditLogger newLogger;
+    private AuditLogger logger;
+
+    @Before
+    public void before() {
+        newLogger = new AuditLogger();
+
+        logger = new AuditLogger();
+        logger.setDevice(new Device());
+    }
+
+    @Test
+    public void reconfigure_UpdatesFacility_WhenCalled() {

```

Comment:
I think test names would benefit from changing "WhenCalled" to specific condition. I know some people say that you should only have one assert per tests, but in this situation I don't this applies. If you are just setting one member other should be null so don't you want to assert That other fields are still null?

I think better approach here would be to all of the fields and assert on all of them. And only have additional tests for branches (if in clearActiveConnectionsAndClose), which I wouldn't really ask for to be honest.